### PR TITLE
feat(jellyfish-api-core): add blockchain getMempoolDescendants RPC

### PR DESCRIPTION
**Pull request by chanyusiang**
#### What this PR does / why we need it:
This PR implements blockchain.getMempoolDescendants() from ain

#### Which issue(s) does this PR fixes?:
Part of #48

### CODE REVIEW

**Review by cwkang1998 — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/src/category/blockchain.ts [hunk 1]
```diff
@@ -225,6 +225,34 @@ export class Blockchain {
   async getChainTxStats (nBlocks?: number, blockHash?: string): Promise<ChainTxStats> {
     return await this.client.call('getchaintxstats', [nBlocks, blockHash], 'number')
   }
+
+  /**
+   * Get all in-mempool descendants for the given transaction as string[]
+   * @param {string} txId the transaction id in mempool
```
Comment (jingyi2811) [inline packages/jellyfish-api-core/src/category/blockchain.ts:231]: Need a break line between the comment and the first line that consist of @param

Comment (jingyi2811) [inline packages/jellyfish-api-core/src/category/blockchain.ts:231]:
```suggestion
   * @param {string} txId the transaction id
```
like how you write in https://github.com/JellyfishSDK/jellyfish/pull/1404/files#diff-401e60eaa7898f5ac9e4d982fcf34c68a2cd450a7a3c372d8243827ba6189c5bR239

**Comment by jingyi2811** [comment]
What happened if you passed in txid that the length is not 64?

File: docs/node/CATEGORIES/01-blockchain.md [hunk 2]
```diff
@@ -392,3 +392,15 @@ interface ChainTxStats {
   txrate: number
 }
 ```
+
+## getMempoolDescendants
+
+Get all in-mempool descendants if txId is in mempool as string[] if verbose is false else as json object.
```
Comment (jingyi2811) [inline docs/node/CATEGORIES/01-blockchain.md:398]: The text here should be the same as the one you wrote in your RPC file

File: packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolDescendants.test.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      'bip125-replaceable': expect.any(Boolean)
+    })
+  })
+
+  it('should return array of txids if verbose is false', async () => {
+    const ancestorTxId = await testing.rpc.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.0003)
+    await waitForExpect(async () => {
+      await testing.rpc.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.0003)
+      const mempoolEntry: string[] = await testing.rpc.blockchain.getMempoolDescendants(ancestorTxId, false)
+      expect(mempoolEntry.length).toBeGreaterThan(0)
```
Comment (jingyi2811) [inline packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolDescendants.test.ts:62]: Maybe can check every element whether it has length = 64 or not

File: packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolDescendants.test.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    expect(mempoolEntry.length).toStrictEqual(0)
+  })
+
+  it('should return JSON object if verbose is true', async () => {
+    const ancestorTxId = await testing.rpc.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.0003)
+    await waitForExpect(async () => {
+      await testing.rpc.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.0003)
+      const mempoolEntry: MempoolTx = await testing.rpc.blockchain.getMempoolDescendants(ancestorTxId, true)
+      expect(mempoolEntry.length).toBeGreaterThan(0)
+    }, 10000)
```
[the remark this answers is not in the document]
Comment (chanyusiang) [reply packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolDescendants.test.ts:32]: I need `waitForExpect` to ensure `mempoolEntry` is not empty though

**Pull request closed without merging**